src/main/webapp/search-interview.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Search for an Interview</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="global.css">
+    <link rel="stylesheet" href="search-interview.css">
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+      integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+      integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+      integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+    <script src="search-interview.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <!-- TODO: add navbar here. -->
+      <h1 class="text-center">Search for an Interview</h1>
+      <h5 class="text-center lead">Search for possible times in which you will be interviewed!</h5>
+      <br>
+      <form>
+        <div class="form-group">
+          <!-- TODO: add parameters for this form. -->
+          <button type="button" class="btn btn-primary" onclick="loadInterviews()">Search</button>
+        </div>
+      </form>
+      <br>      
+      <section hidden id="search-results">
+        <!--TODO: Include Timezone consideration; Add pagination; Render these interview times dynamically;-->
+        <form>
+          <div class="form-row">
+            <div class="col-3 date-label">
+              <label>Monday 6/29</label>
+            </div>
+            <div class="col-5">
+              <select class="form-control">
+                <option>8:15 AM - 9:15 AM</option>
+                <option>10:30 AM - 11:30 AM</option>
+                <option>1:00 PM - 2:00 PM</option>
```

Comment:
Just a note: this display method is fine for now, but I think a more helpful and intuitive one would be to show all the interview times in a calendar grid, like the weekly view in Google Calendar. This will be a more complex interface, so you don't have to change it now, but if it's something you would like to do later (I think it would be worth the time), please add a TODO.

Reply:
One of the problems I see with this is that we will have overlapping events, since we are showing them all possible times. Worst/best-case scenario an event every 15 minutes.

Reply:
That's true but still seems like a solvable problem with some inventive interface design. Definitely more complex than I originally envisioned though.

Reply:
Still don't see how the calendar would have the overlapping events problem, unless we put in lines for the start times instead of blocks... l put the TODO in though, so we can discuss and decide later.